### PR TITLE
added user controller with actions, added name migration to Users

### PR DESCRIPTION
**Pull request by ninjacazaam**
Check controller actions plz

### CODE REVIEW

**Review by scallywag89 — APPROVED** [review]
looks good

**Review by Jasperinabox — APPROVED** [review]
NICE NICE NICE

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,9 @@
 Rails.application.routes.draw do
+  get 'users/new'
```
Comment (ninjacazaam) [inline config/routes.rb:2]: We can delete these since we hae resources

Comment (ambuhlily) [inline config/routes.rb:2]: do not need these extra routes

**Review by ninjacazaam — COMMENTED** [review]
need to change routes file

**Review by ambuhlily — CHANGES_REQUESTED** [review]
remove extra user routes

**Review by ambuhlily — APPROVED** [review]
good

**Review by scallywag89 — APPROVED** [review]
nice one

**Review by Jasperinabox — APPROVED** [review]
ez

**Review by ninjacazaam — COMMENTED** [review]
Lgtm

**Pull request merged**